<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamsPage</class>
 <widget class="QWidget" name="StreamsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="7">
    <widget class="LineEdit" name="search">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="TreeView" name="view"/>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="importStreams"/>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="exportStreams"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="addStream"/>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="removeStream"/>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="editStream"/>
   </item>
   <item row="2" column="6">
    <widget class="QToolButton" name="replacePlaylist"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
